.golangci.yml

```diff
@@ -5,7 +5,7 @@ linters-settings:
   goimports:
     # put imports beginning with prefix after 3rd-party packages;
     # it's a comma-separated list of prefixes
-    local-prefixes: github.com/onflow/flow-go
+    local-prefixes: github.com/onflow/flow-go/
```

Comment:
Previously this also incorrectly considered e.g. `flow-go-sdk` a local package.

This whole line should maybe just be removed instead?